docs/source/tutorials/cord19kg_tutorials/getting_started_cord19kg.rst

```diff
@@ -7,8 +7,45 @@ We recommend starting from the `Co-occurrence analysis tutorial <https://github.
 
```

Comment:
The content here looks the same as https://github.com/BlueBrain/BlueGraph/blob/master/cord19kg/README.rst. How about using include directive so that we maintain this text in once place.

.. include:: ../../../../../cord19kg/README.rst

Reply:
I copy pasted the content explicitly, because otherwise the images are not well rendered (their relative path is not the same from README and from the docs). Here is the issue described: https://stackoverflow.com/questions/50261137/docs-missing-images-when-using-include-directives-rst-sphinx

Reply:
At the moment, I would prefer to keep it here and then figure out the best way to fix this (duplicate images and put them in the same relative path or smth like that)